src/Joint.h

```diff
@@ -416,7 +462,15 @@ inline sva::MotionVecd Joint::tanAccel(const std::vector<double>& alphaD) const
 
 inline std::vector<double> Joint::zeroParam() const
 {
-	return ZeroParam(type_);
+	auto q = ZeroParam(type_);
```

Comment:
Is using `auto` here really necessary ? Well I guess it's entirely determined by the return type of the function anyways.